packages/rehype-stringify/package.json

```diff
@@ -29,10 +29,12 @@
     "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)"
   ],
   "files": [
-    "index.js"
+    "index.js",
+    "types/index.d.ts"
   ],
+  "types": "types/index.d.ts",
   "dependencies": {
-    "hast-util-to-html": "^7.0.0",
+    "hast-util-to-html": "^7.1.1",
```

Comment:
Any reason this is needed? I think I prefer not touching version ranges when not needed. I don’t see a benefit in updating, instead, it may force duplicate packages (if someone were to depend on `7.1.0` somewhere in their tree too), and it creates more noise in lock files?

Reply:
This version includes syntax-tree/hast-util-to-html#20. This is needed so `HastUtilToHtmlOptions` don’t need to be wrapped in `Partial`.